Makefile

```diff
@@ -434,7 +434,7 @@ npm-update: npm-check
 
 .PHONY: js
 js: npm
-	npx eslint public/js
+	npx eslint --ignore-pattern semantic.dropdown.js public/js
```

Comment:
Use `.eslintignore` instead.

Reply:
Or actually, move vendor lib to `public/vendor`, it has no place in `public/js`.

Reply:
@silverwind it's not "exactly" vendor, since it's been specifically modified for Gitea. It's now Gitea code.
One possibility is to move it to its own project @ gitea.com.

Reply:
There's no .eslintignore file?

Reply:
You can create one. I suggest renaming the file to `semantic.dropdown.custom.js` to highlight that it's a customized version and then create `.eslintignore` with content `/ public/js/semantic.dropdown.custom.js`.

Reply:
Ah ok, will try that.

Reply:
Ok, done!

Reply:
Looking good, thanks